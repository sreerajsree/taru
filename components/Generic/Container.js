export default function Container({ children }) {
  return <div className="container 2xl:max-w-screen-xl mx-auto px-5">{children}</div>;
}
