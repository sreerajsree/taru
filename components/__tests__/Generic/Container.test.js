import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import Container from '../../Generic/Container';

describe('Container', () => {
  test('it renders and matches snapshot', () => {
    const { asFragment } = render(<Container />);
    expect(asFragment()).toMatchSnapshot();
  });
});
