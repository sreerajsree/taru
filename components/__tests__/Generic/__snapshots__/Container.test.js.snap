// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container it renders and matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container 2xl:max-w-screen-xl mx-auto px-5"
  />
</DocumentFragment>
`;
